### DISCUSSION TITLE
مهاجرت، اقامت ، تحصیل در خارج ، ثبت استارتاپ و سرمایه گذاری

### DISCUSSION THREAD

**Discussion by MORANLABS** [WeAreMahsaAmini/Core#351] [Ideas]
سلام

لطفا در سایت بخشی جهت خدمات مشاوره برای مهاجرت و اقامت در خارج از ایران اضافه کنید تا افرادی که نمیتوانند در ایران بمانند ما بهشون مشاوره و راهنمائی بدهیم تا بتوانند ا ایران رفته و خارج از سانسور با شرایط بهترین به فعالیت خود ادامه بدهند.

یعنی بین ما و امثال ما و افرادی که تجربه دارند، از یک طرف و افرادی که نیاز به مشاوره و راهنمائی دارند از طرف دیگر ارتباط برقرار بشود.

شرایط خودمان را در لینک زیر توضیح داده ایم:

https://github.com/WeAreMahsaAmini/Core/discussions/348

**Comment by theStratus** [comment]
دوست عزیز البته که هیچ ایده ای بد نیست. به نظرم شما میتونید اطلاعاتی که میفرمایید رو در وبسایتتون قرار بدید و یا به رایگان مشاوره بدید. چون در غیر این صورت این ایده بیشتر معنی تبلیغات میده. 
مضاف بر اینکه این بار صحبت از موندن و ساختن هست. گرچه هر کس در تصمیم گیری مختاره. به نظرم الویت ایده در خصوص وضعیت فعلی، ادمهای داخل ایران و کمک و حمایت به مردمه داخل ایران هست.

**Comment by iranianpep** [comment]
What about blocking those trying to gain personal benefits from this movement?!

**Reply by theStratus** [reply]
I believe the idea of the movement is to let all people to express their opinion freely. Others like us may decide to follow them or not. Blocking people sounds like the government to me. Maybe @MORANLABS decides to help people free of charge. However, this topic is not what I want to follow or promote.

**Reply by iranianpep** [reply]
> Maybe @MORANLABS decides to help people free of charge.

It's not mentioned anywhere the service is going to be free. This looks like an ad to me as opposed to other ideas that genuinely trying to help. It's shifting the focus (Imagine we could spend this time discussing on another constructive idea). That's why I think the user needs to be blocked or the discussion needs to be closed by the admin.

**Reply by AnonymousMaximus2** [reply]
> What about blocking those trying to gain personal benefits from this movement?!

When you say Block the helper, this statement comes to my mind:

May I ask you shut up please? 


you words like some Iranian Government that some times using words to terrorize people, 

Defamation and charater terrorizing is the weapon of dictators.

Iran Government does not interested to people travel or immigrate or go outside of Iran, 

inside Iran people needed as much as outside of Iran needed.

If there are not 8 Million Iranian outside of Iran, 
No one outside of the Iranian borders can hear Iranian voices,

If even they ask for money, it is still helpful for many of Iranians,

even US government does not provides any services for free, here GITHUB provides some services for free, but some services for money to cover its expenses, 

wise people offer some free services and offer some services for money,

Only stupid people offer everything for free that after a while they become bankrupt.

but the free services can not be provided to Iranian QODS Corps Guards that around the world terrorizing the people.

you must provide some formula that clients does not have any relations with Iranian spy and QODS Corps Guards,

**Reply by AnonymousMaximus2** [reply]
> What about blocking those trying to gain personal benefits from this movement?!

در پروفایل تان نام خودتان را عباسی نوشته اید

یاد خبری از خبرگزاریها افتادم:

عباسی نماینده مجلس ایران گفته است که فرزندان مسئولین نخبه و دارای ضریب هوشی بالائی هستند و به ایران باز خواهند گشت و مدیران آینده ایران هستند.

یعنی اینهایی که با پول مردم ایران اختلاس کرده و با حمایت دولت ایران به خارج رفته اند قرار نیست در خارج بمانند و همه نخبه ها را میخواهند داخل ایران زندانی کنند و هر کسی از خودشان است مدیر و اختلاسگر باشد و هر کسی از نخبه ها از خودشان نیست در دانشگاه شریف محاصره کرده و قتل عام کنند نگذارند از ایران فرار کنند ؟

راستی با آن حسن عباسی تند رو که دنیا را تهدید کرده نسبتی ندارید؟

مسئولین دولت ایران خیلی از فرار نخبه ها ترس دارند چون خارج از ایران خیلی دست شان رو میشود و نخبه ها نیز دست شان خیلی برای مبارزه باز میشود

خیلی دلم میخواد جواب شما را بدانم.

**Reply by iranianpep** [reply]
Ok, first of all, mind your language! This is my last message anyway, and I'm not going to waste my time anymore, especially for someone who joined this platform a month ago!

1. I don't have any problem with people leaving the country. In fact, I've been living overseas for years.
2. I'm not saying the service should be free, and I'm not against any migration consultants.

The main problem is **misusing the movement as a way to promote your business** (no matter how good you're business is). Is this that hard to understand?! In any other popular forum like Reddit, if you don't follow the rules, you're simply banned/blocked. This has nothing to do with dictatorship my friend 🙂
